pkg/cluster/spec/instance.go

```diff
@@ -44,6 +44,7 @@ const (
 	ComponentDrainer          = "drainer"
 	ComponentPump             = "pump"
 	ComponentCDC              = "cdc"
+	ComponentTiKVCDC          = "tikv-cdc"
```

Comment:
What's the difference between `cdc` and `tikv-cdc`? This is added to the `cluster/spec` package, so does `tiup-cluster` need to be aware of the new component?

Reply:
`tikv-cdc` is another CDC frame but focus on NoSQL scenario, while `cdc` focus on SQL database. Please refer to https://github.com/tikv/migration/blob/main/cdc/README.md.

`tiup-cluster` need to be aware of the new component as well, please see https://github.com/pingcap/tiup/pull/2022.